src/main/software/amazon/event/ruler/ByteMachine.java

```diff
@@ -91,7 +90,7 @@ Set<NameStateWithPattern> transitionOn(String valString) {
         boolean fieldValueIsNumeric = false;
         if (hasNumeric.get() > 0) {
             try {
-                final double numerically = JavaDoubleParser.parseDouble(valString);
```

Comment:
can we keep our version here, I am pretty sure this one is faster than the standar double parser

Reply:
Hey Rudy, 
that's a good catch. let me double check the benchmarks. I thought JavaDoubleParser was faster too. I'll will respond on Tuesday (away until then).

Reply:
I don't see any regressions or improvements within the benchmarks because most of the time is spent in `ComparableNumber` but when testing in isolation against the numbers within the benchmark data, I see this FastDoubleParser to be slightly better (45 seconds vs 50 seconds). Added it back for now.